smpplib/command.py

```diff
@@ -177,11 +177,19 @@ def _generate_string(self, field):
                 field_value = field_value[0:self.params[field].max - 1]
 
             if field_value:
-                value = field_value + chr(0)
+                if not isinstance(field_value, bytes):
+                    value = field_value + chr(0)
+                else:
+                    value = field_value + chr(0).encode()
             else:
-                value = chr(0)
```

Comment:
`value = six.b(chr(0))` or `six.b('\0')`.  
should be just enough

I also think we've dropped too old python versions, so just `b'\0'` will also work

Reply:
Fine with me :)